### PR TITLE
[json-rpc] update event and account doc

### PR DESCRIPTION
**Pull request by xli**

### CODE REVIEW

File: json-rpc/docs/type_event.md [hunk 1]
```diff
@@ -104,6 +104,46 @@ Event emitted when the url used for off-chain dual attestation checking is rotat
...
+
+Event emitted when a new block is created
+
+| Name          | Type                         | Description                           |
+|---------------|------------------------------|---------------------------------------|
+| type          | string                       | Constant string "newblock"            |
+| round         | unsigned int64               | Round number                          |
+| proposer      | unsigned int64               | proposer account address, hex-encoded |
+| proposed_time | unsigned int64(microseconds) | proposed timestamp                    |
+
```
Comment (tzakian) [inline json-rpc/docs/type_event.md:136]: cc @runtian-zhou to make sure this looks good to him/for help on the description for upgrades

File: json-rpc/docs/type_event.md [hunk 2]
```diff
@@ -104,6 +104,46 @@ Event emitted when the url used for off-chain dual attestation checking is rotat
 | new_base_url                | string | New URL endpoint for off-chain communication            |
 | time_rotated_seconds        | u64    | Blockchain time (in seconds) when the rotation occurred |
 
+#### upgrade
+
+TODO: description
```
Comment (runtian-zhou) [inline json-rpc/docs/type_event.md:109]: I would put "Event emitted when a WriteSet transaction is committed which causes the state to be updated". Thoughts?

File: json-rpc/docs/type_event.md [hunk 3]
```diff
@@ -104,6 +104,46 @@ Event emitted when the url used for off-chain dual attestation checking is rotat
...
+
+| Name      | Type   | Description                 |
+|-----------|--------|-----------------------------|
+| type      | string | Constant string "upgrade"   |
+| write_set | string | TODO: desc                  |
```
Comment (runtian-zhou) [inline json-rpc/docs/type_event.md:114]: lcs serialized writeset.

**Review by thefallentree — APPROVED** [review]

**Pull request closed without merging**